flink-runtime/src/main/java/org/apache/flink/runtime/state/AbstractStateBackend.java

```diff
@@ -66,4 +66,9 @@ public OperatorStateBackend createOperatorStateBackend(
 			String operatorIdentifier) throws Exception {
 		return new DefaultOperatorStateBackend(env.getUserClassLoader());
 	}
+
+	@Override
+	public void close() throws Exception {
+
+	}
```

Comment:
`AbstractStateBackend` has never been part of the public API. Given that, we won't break at least the API stability promise.